routes/users.js

```diff
@@ -1,9 +1,14 @@
-const express = require('express');
-const router = express.Router();
+const express = require('express')
+const User = require('../database/UserSchema.js')
+const router = express.Router()
+const mongoose = require('mongoose')
+const Widget = require('../database/WidgetSchema.js')
```

Comment:
This import is not used.  Also, the filename `WidgetSchema` is very misleading if this module exports a model - I would move model definition modules into a `models` folder, instead of database, and remove the `Schema` part of the file name.  As well, files should be named with lower snake case.